Assets/MixedRealityToolkit-SDK/Inspectors/Input/Handlers/ControllerPoseSynchronizerInspector.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.﻿
+
+using Microsoft.MixedReality.Toolkit.Internal.Definitions.Utilities;
+using Microsoft.MixedReality.Toolkit.SDK.Input.Handlers;
+using UnityEditor;
+
+namespace Microsoft.MixedReality.Toolkit.SDK.Inspectors.Input.Handlers
+{
+    [CustomEditor(typeof(ControllerPoseSynchronizer))]
+    public class ControllerPoseSynchronizerInspector : Editor
```

Comment:
Since this is being renamed as (as far as I could tell from our conversation yesterday) redone slightly differently than AttachToController, can we change the meta's GUID to not prevent porting the original AttachToContoller over at some point?

Reply:
I purposefully kept the GUID the same, but I can regenerate it if you'd like.

Reply:
Done

Reply:
If we think it'll eventually do everything the original script did, we can keep them. It feels like it's going for a slightly different use case though.

Reply:
agreed.

Reply:
Regenerating the guids will help in your "re-port". (Although, I think you should address that in the controller viz in #2492)